jib-gradle-plugin/README.md

```diff
@@ -176,6 +176,7 @@ Field | Type | Default | Description
 `from` | [`from`](#from-closure) | See [`from`](#from-closure) | Configures the base image to build your application on top of.
 `container` | [`container`](#container-closure) | See [`container`](#container-closure) | Configures the container that is run from your built image.
 `extraDirectories` | [`extraDirectories`](#extradirectories-closure) | See [`extraDirectories`](#extradirectories-closure) | Configures the directories used to add arbitrary files to the image.
+`containerizingMode` | `String` | `exploded` | If set to `packaged`, puts the JAR artifact built by the Gradle Java plugin into the final image. If set to `exploded` (default), containerizes individual `.class` files and resources files.
```

Comment:
Should it be "Gradle Jar plugin" not "Gradle Java plugin"?

Reply:
Not totally sure, but I thought the "Java Plugin" just provides the `jar` task. https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_tasks

Reply:
Ah ok, I wasn't sure.